docs/placeholder.md

```diff
@@ -1,2 +1,380 @@
 
-this is a placeholder
+# DE Africa Water Quality Pipeline Documentation
+
+## Overview
+
+The DE Africa Water Quality Pipeline is a workflow that processes satellite imagery to generate annual water quality variables across Africa. The pipeline analyzes multi-sensor satellite data to produce metrics like total suspended solids (TSS), chlorophyll-a concentration, optical water types, and water frequency statistics.
+
+## Prerequisites
+
+### Environment Setup
+
+```bash
+# Clone the repository
+git clone <deafrica_water_quality>
+
+# Install in editable mode
+cd deafrica_water_quality
+pip install -e .
+
+# Restart kernel if working on DE Africa Sandbox
+```
+
+### Requirements
+- Access to DE Africa Sandbox datacube environment
+- Python environment with required dependencies
+- YAML configuration file for analysis parameters
+- Appropriate filesystem access (local or cloud storage)
+
+---
+
+## Pipeline Architecture
+
+### Stage 1: Task Generation (`wq-generate-tasks`)
+
+Generates a list of processing tasks based on temporal range, frequency, and spatial tiles.
+
+#### Command
+
+```bash
+!wq-generate-tasks [OPTIONS] TEMPORAL_RANGE FREQUENCY OUTPUT_FILE
+```
+
+#### Arguments
+- **TEMPORAL_RANGE**: Time period in ISO format (e.g., `2020-05--P1M` for May 2020)
+- **FREQUENCY**: Temporal binning interval
+  - Supported: `annual`, `semiannual`, `monthly`, `fortnightly`, `weekly`
+- **OUTPUT_FILE**: Path to write task list (text file with one task per line)
+
+#### Options
+- `--tile-ids`: Comma-separated tile IDs (e.g., `x188/y109,x178/y095`)
+- `--tile-ids-file`: Path to file containing tile IDs
+- `--place-name`: Predefined test area name (view with `wq-list-test-areas`)
+
+**Note**: Specify exactly one of `--tile-ids`, `--tile-ids-file`, or `--place-name`. If none specified, generates tasks for all Africa tiles.
+
+#### Temporal Range Behavior
+If the temporal range doesn't completely cover a temporal bin, all data for the complete bin will be loaded:
+- `2025-08-13--P3D` with `weekly` frequency → loads `2025-08-13--P1W`
+- `2025-08-13--P1M` with `monthly` frequency → loads `2025-08--P1M` and `2025-09--P1M`
+
+#### Output
+Text file containing task IDs in format: `{temporal_id}/x{xx}/y{yy}`
+
+Example: `2015--P1Y/x200/y034`
+
+---
+
+### Stage 2: Water Quality Processing (`wq-process-annual-wq-variables`)
+
+Processes tasks to generate water quality variables and outputs as Cloud Optimized GeoTIFFs (COGs).
```

Comment:
Need to specify the water quality variables generated here are annual to differentiate them from the shorter timeseries variables in future